testnet-node/docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: '2.2'
+services:
+  testnet-node-core:
+    build:
+      context: ..
+      dockerfile: Dockerfile
+    # image: radixdlt/babylon-node:rcnet-v1-c6360105d-build-2
+    init: true
+    ulimits:
+      nofile:
+        soft: 65536
+        hard: 65536
+      memlock: -1
+    # restart: unless-stopped
```

Comment:
Should this be removed? Or if it's an option that developers might want to use, we should add a comment explaining why they might want to change this